styles/base.css

```diff
@@ -228,92 +194,30 @@ header {
     width: 5.5rem;
 }
 
-/* Main */
-
-main {
-    flex: 1 0 auto;
-}
-
 /* Footer */
 
-.divider-line {
-    border-top: 1px solid #545454;
-    margin: 10px -16px 10px;
-}
-
-@media (min-width: 992px) {
-    .divider-line {
-        margin: 20px -30px 20px;
-    }
-
-}
```

Comment:
This is now `border-light border-top` using `bs-light-rgb` color. See comment above.